scripts/postalcode.cs.template

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Text.RegularExpressions;
+using PostalCodes.GenericPostalCodes;
+
+namespace PostalCodes
+{
+	internal class @@CountryCode@@PostalCode : AlphaNumericPostalCode
```

Comment:
That's cool idea! I'll do that :)
On Mar 23, 2015 2:46 PM, "Rafal Nowosielski" <notifications@github.com>
wrote:
>
> In scripts/postalcode.cs.template:
>
> > @@ -0,0 +1,102 @@
> > +using System;
> > +using System.Text.RegularExpressions;
> > +using PostalCodes.GenericPostalCodes;
> > +
> > +namespace PostalCodes
> > +{
> > + internal class @@CountryCode@@PostalCode : AlphaNumericPostalCode
>
> It would be cool if the generated classes were partial classes. It would
always make it easier to add extra overridden (non generated) functionality
to the exotic zip codes.
>
> —
> Reply to this email directly or view it on GitHub.